crm_partner_company_group/hooks.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+
+def pre_init_hook(env):
+    """Prepopulate stored related fields for faster installation"""
+    logger = logging.getLogger(__name__)
+    logger.info("Prepopulating stored related fields")
+    env.execute(
+        """
+        ALTER TABLE crm_lead
+        ADD COLUMN IF NOT EXISTS company_group_id integer;
```

Comment:
Not necessary anymore since https://github.com/odoo/odoo/pull/65232, although I'm slightly wondering why this code does not actually populate the values. Is that on purpose?

Reply:
Thanks @StefanRijnhart, you're right. We don't need this hook anymore. I just removed it.